nixos/modules/services/network-filesystems/ipfs-cluster.nix

```diff
@@ -0,0 +1,123 @@
+{ config, lib, pkgs, options, ... }:
+with lib;
+let
+  cfg = config.services.ipfs-cluster;
+  opt = options.services.ipfs-cluster;
+
+  # secret is by envvar, not flag
+  initFlags = toString [
```

Comment:
Maybe `initFlags = optionalString (cfg.initPeers != []) "--peers ${lib.concatStringsSep "," cfg.initPeers}";`?

Reply:
I figured there would be more flags soon enough